### ISSUE TITLE
MPS (meters per second) support

### ISSUE THREAD

**Issue by mwgg** [keesL/metar#3]
Hi,

The program should support wind in meters per second, which is common for Russia and some other countries. Currently this is causing erroneous output:

```
UUEE 231300Z 27006MPS 9999 SCT049 08/M02 Q1019 R24R/CLRD62 R24L/CLRD62 NOSIG
...
Parsing token `27006MPS'
   Unmatched token = 27006MPS
...
Wind direction: 0 (N)
Wind speed    : 0
Wind gust     : 0
```

My regex skill aren't up to par, otherwise I would help out. Thanks!

**Comment by keesL** [comment]
Thank you for the suggestion; I was unaware of the practice. I'll add it to my TODO list, but due to workload constraints, I cannot promise that I'll get around to it soon.

**Comment by keesL** [comment]
Metar will now recognize mps as a valid unit for wind speeds.

**Comment by keesL** [comment]
closes #3
